### PR TITLE
Voice management module's code cleaning

### PR DESCRIPTION
**Pull request by DrumSlayers**
-Cleaned code following @M0NsTeRRR  recommandations
-Bitrate is now fetched from config.toml

### CODE REVIEW

File: extensions/management.py [hunk 1]
```diff
@@ -25,35 +25,36 @@ async def on_voice_state_update(self, member, before, after):
...
-            for i in count(1):
+            new_channel_name = None
+            channel_number = 1
+
+            while new_channel_name is None:
                 if (
-                    not self.bot.config["voice_channel_creation"]["new_channel_name"]
-                    + " #"
-                    + str(i)
+                    not f'{self.bot.config["voice_channel_creation"]["new_channel_name"]} #{channel_number}'
```
Comment (M0NsTeRRR) [inline extensions/management.py:45]: create a variable with this value to use it in the if and variable assignation

**Review by M0NsTeRRR — CHANGES_REQUESTED** [review]

**Review by M0NsTeRRR — APPROVED** [review]

**Comment by DrumSlayers** [comment]
Close #70

**Comment by DrumSlayers** [comment]
Close issue #70

**Comment by DrumSlayers** [comment]
Close #70

**Pull request merged**